cmd/horcrux/cmd/state.go

```diff
@@ -0,0 +1,216 @@
+package cmd
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/strangelove-ventures/horcrux/signer"
+)
+
+func init() {
+	pvCmd.AddCommand(showPvCmd())
+	pvCmd.AddCommand(resetPvCmd())
+	stateCmd.AddCommand(pvCmd)
+
+	shareCmd.AddCommand(showShareCmd())
+	shareCmd.AddCommand(resetShareCmd())
+	stateCmd.AddCommand(shareCmd)
```

Comment:
I think you can merge these two. I don't see a usecase for individually setting them so lets make them just 1 command to set the state for both.

Reply:
Also lets make the `--height` a required arg as opposed to a flag.

Reply:
Gotcha, I've changed it to the following:
```bash
horcrux
    state
        show
        set [height]
```

Reply:
This is awesome!